tests/settings/SectionTest.py

```diff
@@ -204,3 +206,52 @@ def test_set_default_section(self):
         sections['all.c.codestyle'].set_default_section(sections)
         self.assertEqual(sections['all.c.codestyle'].defaults,
                          sections['all'])
+
+    def test_extract_aspects_from_section(self):
+        section = Section('section')
+        # No aspects setting
+        self.assertIsNone(extract_aspects_from_section(section))
+
+        section.append(Setting(
+            'aspects',
+            'spelling, commitmessage, methodsmell'))
+        section.append(Setting(
```

Comment:
The first test method shouldnt use `excludes`.  A second test method should use `excludes`.